Source/Core/Core/PowerPC/Jit64/Jit_SystemRegisters.cpp

```diff
@@ -282,18 +282,33 @@ void Jit64::mfspr(UGeckoInstruction inst)
     // no register choice
 
     gpr.FlushLockX(RDX, RAX);
+    gpr.FlushLockX(RCX);
+
+    const void* rcx_relative_base = &CoreTiming::g_last_OC_factor_inverted;
+    MOV(64, R(RCX), ImmPtr(rcx_relative_base));
+
+    // We're relying on these globals being placed close (within +/-2GB) to each other in memory.
+    const auto MRelative = [rcx_relative_base](void* ptr) -> OpArg {
```

Comment:
I'd imagine the lambda return type would be inferred in this case, would it not?